inc/blocks/ab-test/edit.css

```diff
@@ -0,0 +1,13 @@
+button.ab-test-xb-preview__tab {
```

Comment:
The `button` part of the selector isn't strictly necessary. Also consider setting a default font-family, it will mirror whatever's on the site - you can look at how the font-family is set for the admin bar for that, think it might be via the `.wp-core-ui` class.

If you're having issues with styling and CSS specificity for something like this you could use `!important` - it's generally frowned on but this is a fair use-case for it.